std/complex.d

```diff
@@ -665,14 +669,18 @@ unittest
 
 
 /++
-   Calculates the squared modulus of a complex number.
-   For genericity, if called on a real number, $(D sqAbs) returns its square.
+   Params:
+    z = A complex number.
+    x = A real number.
```

Comment:
Two overloads are merged here, so the parameters of both overloads need to be documented.